### ISSUE TITLE
VAE Training Code

### ISSUE THREAD

**Issue by yukiarimo** [ace-step/ACE-Step-1.5#44]
Hello!

Can we please get a VAE training code? (At least, the decoder part)

Thanks!

**Comment by ChuxiJ** [comment]
see: https://github.com/Stability-AI/stable-audio-tools

**Comment by yukiarimo** [comment]
Oh, really? It’s based off of this? Can you please give some training/architecture hyper parameters so it will be easier to reverse-engineer this? Thanks!

**Comment by yukiarimo** [comment]
@ChuxiJ So, because we do not have both the generator and discriminator, but one checkpoint, how I can fine-tune and calculate the loss? Can you give me a script for training ACE-Studio-1.5's VAE directly?

**Comment by yukiarimo** [comment]
Don’t close

**Comment by ChuxiJ** [comment]
@yukiarimo https://huggingface.co/ACE-Step/ace-step-v1.5-1d-vae-stable-audio-format
